src/gridlayout.jl

```diff
@@ -1,45 +1,48 @@
-GridLayout(; kwargs...) = GridLayout(1, 1; kwargs...)
+GridLayout(; @nospecialize(kwargs...)) = GridLayout(1, 1; kwargs...)
 
 observablify(x::Observable) = x
-observablify(x, type=Any) = Observable{type}(x)
+observablify(x) = Observable(x)
+struct Observablify{T} end
+Observablify{T}(x) where T = convert(Observable{T}, x)::Observable{T}
```

Comment:
what is the purpose of this struct? isn't it weird to have a constructor `Observablify` that creates an `Observable`?

Reply:
Yeah, I'll change this. I can't remember right now why a regular function call didn't seem adequate.